src/scss/attendee/variables.scss

```diff
@@ -0,0 +1,38 @@
+$sparkle-theme: rgb(99, 42, 199);
```

Comment:
Please, don't add this file back to the code again. There are other ones like `theme.scss`, `layout.scss` etc. that already work well enough by keeping a file like the old `constants.scss` getting too big to maintain.

Reply:
removed [04e9125](https://github.com/sparkletown/sparkle/pull/2840/commits/04e91251fb1d5bfc42be24e1adab66087443d930)